geoips/xarray_utils/data.py

```diff
@@ -668,15 +689,60 @@ def sector_xarrays(
     """
     import numpy
 
+    Parameters
```

Comment:
@evrose54 I think this is where your black error is coming from - these lines are outside the """ """ string (they should go before the import numpy?)

Reply:
Fixed this; issue happened when some other changes got merged into this branch.